### PR TITLE
styled root page

### PR DESCRIPTION
**Pull request by amelia2802**
# ⭐ Descripción
- Styled the root page according to the provided figma web design.
- fixed issue #1 
### Preview
[Advice Generator App.webm](https://user-images.githubusercontent.com/49182604/226535376-0a97855a-fe91-41d3-b4d9-5193ac6852f2.webm)

*Por favor, incluye una breve descripción de tus cambios.*

## 🚀 Tipo de cambio

- [ ] Arreglo de bug (cambios que no rompen nada y resuelven un problema).
- [x] Nueva característica (cambios que no rompen nada y agregan funcionalidad).
- [ ] Cambio radical (arreglo o característica que podría causar que una funcionalidad no funcione como se espera).
- [ ] Este cambio requiere actualización de documentación.

# 📝 Lista de verificación:
(*) -> obligatorio.

- [x] He creado mis propias clases / ids y he comprobado que no afectan a otros diseños de la comunidad *.
- [x] He realizado una auto-revisión de mi propio código *.
- [x] He comentado mi código, particularmente en las áreas difíciles de entender.
- [x] Mis cambios no generan nuevos avisos *.
- [x] No he instalado nuevos módulos con `npm install` o `yarn install` *.
- [x] La solución es de mi autoría y no he copiado código de otros participantes *.
- [ ] No modifiqué ningún otro archivo más que los míos *.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +49,18 @@
   "lint-staged": {
     "*.{astro,cjs,js,jsx,mjs}": "eslint --cache --fix",
     "*": "prettier --cache --ignore-unknown --write"
-  }
+  },
+  "description": "Necesitarás tener instalado [Visual Studio Code](https://code.visualstudio.com/download), [Node.js 16+](https://nodejs.org/en/download/) y la extensión de Astro, la puedes encontrar [aquí](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)",
```
Comment (nujovich) [inline package.json]: You can delete lines 52 from line 65! <3

**Review by nujovich — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (amelia2802) [reply package.json]: Done

**Comment by nujovich** [comment]
@gi-manzano it is up to you!!

**Review by nujovich — APPROVED** [review]

**Comment by gi-manzano** [comment]
@amelia2802 Hello the subtitle, could you use another font? an Inert style would be fine, another point to consider, could you remove the borders of the rocket and not put any type of effects on it. keep it simple
As a final point, I would also remove the movement of the body, leave it static and put some effect like a hover when hovering over the images. Thank you!!!

**Comment by amelia2802** [comment]
> leave it static and put some effect like a hover when hovering over the images. Thank you!!!

@gi-manzano Hello, thank you for providing me with the instructions. As I work on implementing the hover effect for the images, I would like to clarify if you have any particular hovering effect in mind that you would like me to incorporate. This will help me ensure that the final result meets your expectations. Thank you.

**Comment by nujovich** [comment]
> > leave it static and put some effect like a hover when hovering over the images. Thank you!!!
> 
> @gi-manzano Hello, thank you for providing me with the instructions. As I work on implementing the hover effect for the images, I would like to clarify if you have any particular hovering effect in mind that you would like me to incorporate. This will help me ensure that the final result meets your expectations. Thank you.

@amelia2802 could you please update preview son we can take a look to your final changes ?

**Comment by amelia2802** [comment]
> @amelia2802 could you please update preview son we can take a look to your final changes ?

Sure! Here it is
# Preview[Update:]

https://user-images.githubusercontent.com/49182604/232831324-3a36053b-73ca-413b-bdd7-052816964bb1.mp4

@nujovich  should I update the PR?

**Comment by nujovich** [comment]
> > @amelia2802 could you please update preview son we can take a look to your final changes ?
> 
> Sure! Here it is
> 
> # Preview[Update:]
>  Advice.Generator.App.mp4 
> @nujovich should I update the PR?

Hi @amelia2802 ! It is not necessary. Could you please update the logo to have a radius style ?? Other than that, I really love the progress you've made!

**Review by nujovich — APPROVED** [review]

**Review by nujovich — APPROVED** [review]

**Review by nujovich — APPROVED** [review]

**Comment by amelia2802** [comment]
Thank you for giving me the opportunity to contribute to Mermelada Tech! I'm looking forward to contributing more to this repository. 😊

**Pull request merged**